src/dll/Hooks/ValidateScripts.cpp

```diff
@@ -37,7 +36,8 @@ bool _ScriptValidator_Validate(uint64_t self, uint64_t a1, RED4ext::ScriptReport
         }
     }
 
-    auto message = WritePopupMessage(errors);
+    auto& incompatiblePlugins = App::Get()->GetPluginSystem()->GetIncompatiblePlugins();
+    auto message = WritePopupMessage(errors, incompatiblePlugins);
     if (!message.empty())
     {
         MessageBoxA(0, message.c_str(), "Script Validation Error", MB_OK | MB_ICONERROR);
```

Comment:
Think you should use MessageBoxW here and use std::wstring for the message for utf8 reasons we discussed before. i can see you try to basically do the same thing as you did before here with the path unknowingly at least...

I believe fmt uses utf8 strings, but messageboxa as name suggests uses ascii code page which as we discussed may not be utf8 on windows.